src/applications/mhv/secure-messaging/tests/e2e/fixtures/draftsResponse/multi-draft-response.json

```diff
@@ -0,0 +1,228 @@
+        "attachment": null,
+        "sentDate": "2023-11-28T22:34:44.000Z",
+        "senderId": 251391,
+        "senderName": "MHVDAYMARK, MARK",
+        "recipientId": 678848,
+        "recipientName": "DIMITAR TRIAGE GROUP_UAT2929",
+        "readReceipt": null,
+        "triageGroupName": "DIMITAR TRIAGE GROUP_UAT2929",
+        "proxySenderName": null,
+        "threadId": 3103881,
+        "folderId": -1,
+        "messageBody": "Draft 3s testing autosaved testing again manual save. news redux\nSending now",
+        "draftDate": null,
+        "toDate": null,
+        "hasAttachments": false
+      },
+      "links": {
+        "self": "https://staging-api.va.gov/my_health/v1/messaging/messages/3161671"
+      }
+    },
+    {
+      "id": "3103905",
+      "type": "message_thread_details",
+      "attributes": {
+        "messageId": 3103905,
+        "category": "TEST_RESULTS",
+        "subject": "Multiple drafts from legacy",
+        "body": "M2",
+        "attachment": null,
+        "sentDate": "2023-10-31T19:08:27.000Z",
+        "senderId": 251391,
+        "senderName": "MHVDAYMARK, MARK ",
+        "recipientId": 678848,
+        "recipientName": "DIMITAR TRIAGE GROUP_UAT2929",
+        "readReceipt": null,
+        "triageGroupName": "DIMITAR TRIAGE GROUP_UAT2929",
+        "proxySenderName": null,
+        "threadId": 3103881,
+        "folderId": -1,
+        "messageBody": "M2",
+        "draftDate": null,
+        "toDate": null,
+        "hasAttachments": false
+      },
+      "links": {
+        "self": "https://staging-api.va.gov/my_health/v1/messaging/messages/3103905"
+      }
+    },
+    {
+      "id": "3103891",
+      "type": "message_thread_details",
+      "attributes": {
+        "messageId": 3103891,
+        "category": "TEST_RESULTS",
+        "subject": "Multiple drafts from legacy",
+        "body": "P1",
+        "attachment": null,
+        "sentDate": "2023-10-31T19:03:25.000Z",
+        "senderId": 251391,
+        "senderName": "MHVDAYMARK, MARK ",
+        "recipientId": 678848,
+        "recipientName": "DIMITAR TRIAGE GROUP_UAT2929",
+        "readReceipt": null,
+        "triageGroupName": "DIMITAR TRIAGE GROUP_UAT2929",
+        "proxySenderName": null,
+        "threadId": 3103881,
+        "folderId": -1,
+        "messageBody": "P1",
+        "draftDate": null,
+        "toDate": null,
+        "hasAttachments": false
+      },
+      "links": {
+        "self": "https://staging-api.va.gov/my_health/v1/messaging/messages/3103891"
+      }
+    },
+    {
+      "id": "3103882",
+      "type": "message_thread_details",
+      "attributes": {
+        "messageId": 3103882,
+        "category": "TEST_RESULTS",
+        "subject": "Multiple drafts from legacy",
+        "body": "\r\nM1\r\n\r\n\r\nDimitar\r\nMD",
+        "attachment": null,
+        "sentDate": "2023-10-31T19:02:00.000Z",
+        "senderId": 678797,
+        "senderName": "TZVETANOV, DIMITAR  A",
```

Comment:
@fazilqa @haniasma let's try to scrape off any PII from mock data here and in other places if any.

Reply:
done